<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 63, 63)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(27, 170, 222);
border-radius: 2px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="opencarrier_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color:white;
border:0px;}

QPushButton::hover{
color:blue;
}
</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>158</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;</string>
     </property>
     <property name="text">
      <string>Select Carrier</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(27, 170, 222);
border-radius: 2px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="extraction_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;</string>
     </property>
     <property name="text">
      <string>Save As</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color:white;
border:0px;}

QPushButton::hover{
color:blue;
}
</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>460</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="extract_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>760</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: white;
color:black;
}
QPushButton::hover{
color:blue;
}
</string>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>881</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/rabbit.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="password_groupbox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(27, 170, 222);
border-radius: 2px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;</string>
     </property>
     <property name="text">
      <string>Decryption</string>
     </property>
    </widget>
    <widget class="QPushButton" name="decrypt_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color:white;
border:0px;}

QPushButton::hover{
color:blue;
}
</string>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/photos-icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/unlock-icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/white-folder-icon.png</pixmap>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>extract_button</zorder>
   <zorder>password_groupbox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../PycharmProjects/lab/res.qrc"/>
 </resources>
 <connections/>
</ui>
